include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) Intel Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef _ONEDPL_SERIAL_NUMERIC_IMPL_H
+#define _ONEDPL_SERIAL_NUMERIC_IMPL_H
+
+#if (__cplusplus < 201703L) || (defined(__GLIBCXX__) && __GLIBCXX__ < 20200312)
```

Comment:
Hi, @rarutyun 
The reason why we check __GLIBCXX__ is we want to distinguish among libstdc++-9.1, libstdc++-9.2 and lisbtdc++-9.3
std::reduce was added since libstdc++-9.3, so we must provide our own implementation if libstdc++9.1/9.2 is in use.